llama-index-core/llama_index/core/instrumentation/__init__.py

```diff
@@ -31,3 +39,38 @@ def get_dispatcher(name: str = "root") -> Dispatcher:
     )
     root_manager.add_dispatcher(new_dispatcher)
     return new_dispatcher
+
+
+class DispatcherSpanMixin(ABC):
+    """
+    Apply dispatcher.span decorator to implementations of abstract methods
+    as well as any previously decorated methods that have been overridden.
```

Comment:
Could we add an example of the latter in this doc string.

"e.g., ..."

Reply:
I've rewritten the sentence for more clarity, and an example is provided by the unit test.

Reply:
Thank you — the updated comment is very clear!